### PR TITLE
feat: Create the pod-adapter adn pod-controller in knode

### PR DESCRIPTION
**Pull request by OrangeBao**
/cc @xyz2277 
/cc @wangyizhi1 
/cc @wuyingjun-lucky

### CODE REVIEW

File: pkg/knode-manager/adapters/k8s/pod.go [hunk 1]
```diff
@@ -441,7 +419,7 @@ func (p *PodAdapter) createServiceAccount(ctx context.Context, secret *corev1.Se
 }
 
 func (p *PodAdapter) Create(ctx context.Context, pod *corev1.Pod) error {
-	if pod.Namespace == "kube-system" {
+	if pod.Namespace == SkipNamespace {
```
Comment (wuyingjun-lucky) [inline pkg/knode-manager/adapters/k8s/pod.go]: what does Skipnamespace mean？
I prefer reserverdNS

Comment (OrangeBao) [reply pkg/knode-manager/adapters/k8s/pod.go]: ok, i will replace `Skipnamespace` with `ReserverdNS`

Comment (wuyingjun-lucky) [reply pkg/knode-manager/adapters/k8s/pod.go]: I may make a mistake
`reserved` not `reserverd`

File: pkg/knode-manager/knode.go [hunk 2]
```diff
@@ -53,35 +61,35 @@ func NewKnode(ctx context.Context, knode *kosmosv1alpha1.Knode, cmdConfig *confi
...
-	cmInformer := informer.Core().V1().ConfigMaps()
-	secretInformer := informer.Core().V1().Secrets()
-	serviceInformer := informer.Core().V1().Services()
+	cInformer := kubeinformers.NewSharedInformerFactory(client, 0)
+	cPodInformer := cInformer.Core().V1().Pods()
+	cNsInformer := cInformer.Core().V1().Namespaces()
+	cNodeInformer := cInformer.Core().V1().Nodes()
+	cCmInformer := cInformer.Core().V1().ConfigMaps()
+	cSecretInformer := cInformer.Core().V1().Secrets()
+	cServiceInformer := cInformer.Core().V1().Services()
```
Comment (xyz2277) [inline pkg/knode-manager/knode.go]: what's meaning of c?

File: pkg/knode-manager/adapters/k8s/pod.go (hunk 1 above)
Comment (OrangeBao) [reply pkg/knode-manager/adapters/k8s/pod.go]: updated

File: pkg/knode-manager/knode.go (hunk 2 above)
Comment (OrangeBao) [reply pkg/knode-manager/knode.go]: cxxx => client  : the client for member cluster
mxx => master: the client for host cluster

File: pkg/knode-manager/knode.go [hunk 3]
```diff
@@ -97,8 +105,21 @@ func NewKnode(ctx context.Context, knode *kosmosv1alpha1.Knode, cmdConfig *confi
 		return nil, err
 	}
 
+	eb := record.NewBroadcaster()
+	eb.StartLogging(klog.Infof)
```
Comment (wangyizhi1) [inline pkg/knode-manager/knode.go:130]: What is the use of Broadcaster?

Comment (OrangeBao) [reply pkg/knode-manager/knode.go:130]: Create a log record for `kubectl describe xxxxx`

Comment (OrangeBao) [reply pkg/knode-manager/knode.go:130]: https://xieys.club/kubernetes-events/

File: pkg/knode-manager/knode.go (hunk 2 above)
Comment (OrangeBao) [reply pkg/knode-manager/knode.go]: updated

**Pull request merged**